.gitignore

```diff
@@ -3,5 +3,6 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+.DS_Store
```

Comment:
this maybe should go in `.gitignoreglobal`, but if the content writers are using VSCode or something, we'll want to make it easier for them.

Reply:
Good tip.  I've added to my global ignore file as well.